config/search/config.json

```diff
@@ -201,19 +201,25 @@
       }
     },
     {
-      "type": "case",
+      "type": "decisionmaking-flow",
       "on_path": "cases",
```

Comment:
Specifieke reden waarom het path hier niet mee aangepast is? Zou dat wel doen om verwarring te vermijden.

Reply:
Niet echt, was gewoon al zo. Weet niet of Michaël daar een specifieke reden voor had, of dat gewoon was vergeten aan te passen.